app/Models/Config.php

```diff
@@ -67,10 +72,6 @@ public function getConfigValueAttribute($value)
 
     public function setConfigValueAttribute($value)
     {
-        if (is_bool($value)) {
-            $this->attributes['config_value'] = $value ? 'true' : 'false';
-        } else {
-            $this->attributes['config_value'] = $value;
-        }
+        $this->attributes['config_value'] = serialize($value);
```

Comment:
Any reason we can't use
```php
protected $casts = [
        'config_value' => 'array',
    ];
```
Or maybe I misunderstood how it works

Reply:
That would work, but we want to support old values when we decode from the database.  Otherwise, we could just add a migration to convert all existing db values...